src/app/profile/profile.controller.js

```diff
@@ -12,12 +14,11 @@
 
         $scope.usersCollection = [];
         $scope.topReceivers = [];
-        $scope.showLoader = true;
         $scope.buttonDisabled = true;
         $scope.receivedAcorns = false;
         $scope.sentAcorns = true;
         $scope.giveAcorns = false;
-        $scope.received = true;
+        $scope.new = true;
```

Comment:
What does new mean? Variables should define what they are, this is not very definitive.